python-code/set-sample/set-sample1.py

```diff
@@ -1,2 +1,3 @@
 import os
-import math
```

Comment:
This is a  Unused import math
```suggestion 
```

 from our GitHub Enhacement Bot